src/LogNormal/LogNormalSolver.sol

```diff
@@ -279,7 +265,6 @@ contract LogNormalSolver {
                 endReserves.ry = getNextReserveY(
                     poolId, endReserves.rx, endReserves.L, approxPrice
                 );
-                endReserves.ry += 1;
```

Comment:
why delet?

Reply:
Because we are enforcing that the reserves leave us with an invariant >= 0 already.  Incrementing these values by 1 won't do anything from a security pov imo.

Reply:
This was the core purpose of this PR, to solve dx/dy/dL such that we always land at an invariant >= 0.

Reply:
makes sense